source/client-dependencies.txt

```diff
@@ -1,18 +1,271 @@
 .. _client-authentication:
 
-======================
-Authentication Plugins
-======================
+==============
+Authentication
+==============
 
 .. default-domain:: mongodb
 
-The |bi| offers authentication plugins allowing you to authenticate
-using either the ``SCRAM-SHA-1``, ``SCRAM-SHA-256``, or ``PLAIN``
-authentication mechanisms. If you are using the
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+The |bi| supports the following authentication mechanisms for
+authenticating with MongoDB and for authenticating connecting users:
+
+- ``SCRAM-SHA-1``
+- ``SCRAM-SHA-256``
+- ``PLAIN`` (LDAP)
+- ``GSSAPI`` (Kerberos)
+
+The |bi-short| (:binary:`~bin.mongosqld`) uses admin credentials to
+authenticate with MongoDB and :ref:`sample data <bi-cached-sampling>` to
+generate the |bi-short|'s schema. Once authenticated,
+:binary:`~bin.mongosqld` passes connecting client credentials to
+MongoDB for authentication and will display and restrict data according
+to the connecting user's permissions.
+
+The following sections will guide you through configuring the |bi-short|
+to authenticate with your MongoDB deployment, configuring authentication
+of users connecting to the |bi-short|, and how usernames should be
+formatted from your BI tool using authentcation options.
+
+MongoDB Authentication
+----------------------
+
+If authentication is enabled on your MongoDB deployment, you must
+configure the |bi-short| to use the required authentication mechanism
+and provide admin user credentials. The admin user credentials must be
+:manual:`authorized </core/authorization>` to view the superset of data
+you wish to query using the |bi-short|.
+
+.. tabs-auth::
+
+   tabs:
+     - id: uidpwd
+       content: |
+
+         .. note::
+            For information on how to configure |scram| on your MongoDB
+            deployment, see :manual:`SCRAM </core/security-scram>`.
+
+         To enable Challenge and Response (``SCRAM-SHA-1``) as your 
+         |bi-short| authentication mechanism for MongoDB, you must add 
+         the following settings to your 
+         :doc:`mongosqld </reference/mongosqld>` configuration file:
+
+         .. code-block:: yaml
+
```

Comment:
I'm not a fan of the yaml code-block scheme, because it treats quoted strings differently from unquoted strings, and I prefer yaml files in which all the values are the same color. But that's a pet peeve of mine; I don't think we have an official policy on which code-block type to use for yaml files.

Reply:
Fair enough, let me think about this while it gets tech reviewed.